build.gradle.kts

```diff
@@ -6,7 +6,7 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
 plugins {
-    `kotlin-dsl`
```

Comment:
How come it worked so far? Is it a coincidence? luck? or just the warning was there all along?

Reply:
It's worked fine up to now, but when you declared the explicit version for `kotlin-dsl` in #258, that's when it started complaining.

The intention of removing it in this PR is to unblock upgrades to the Gradle version used to build the project.